packages/react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gadgetinc/react",
-  "version": "0.15.15",
+  "version": "0.15.16",
```

Comment:
Maybe don’t bump the version and we can just let this go out with the next version bump